examples/log-bucket/project/.terraform.lock.hcl

```diff
@@ -0,0 +1,38 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
+
+provider "registry.terraform.io/hashicorp/google" {
```

Comment:
since this is an example, we shouldn't check this in. You can add it to the gitignore